department-of-veterans-affairs/abd-vro
# MCP-2218 Add category for conditions

## What was the problem?

Sufficiency decision should only consider encounter-diagnosis conditions. Previously `automatedClaim` route was not sending conditions from lighthouse to downstream services. 

As per design request, if there is no date we should leave the field blank or write "no associated date" on the PDF. 

Associated tickets or Slack threads:

- [MCP-2218](https://amida.atlassian.net/browse/MCP-2218)

## How does this fix it?[^1]


## How to test this PR
- Step 1 audit messages going through automatedClaim routes.
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
